app/src/features/projects/components/ProjectFundingItemForm.tsx

```diff
@@ -71,17 +126,34 @@ export interface IProjectFundingItemFormProps {
  * @param {*} props
  * @return {*}
  */
-
 const ProjectFundingItemForm: React.FC<IProjectFundingItemFormProps> = (props) => {
   const formikProps = useFormikContext<IProjectFundingFormArrayItem>();
-
   const { values, touched, errors, handleChange, handleSubmit, setFieldValue } = formikProps;
-
   // Only show investment_action_category if certain agency_id values are selected
   // Toggle investment_action_category label and dropdown values based on chosen agency_id
   const investment_action_category_label =
     (values.agency_id === 1 && 'Investment Action') || (values.agency_id === 2 && 'Investment Category') || null;
 
+  const findItemLabel = (id: number, type: FundingSourceType) => {
+    return props.sources.find((item) => item.value === id && item.type === type)?.label;
+  };
+
+  // find label for initial value
+  const mapInitialValue = (
+    formValues?: IProjectFundingFormArrayItem
+  ): IAutocompleteFieldOptionWithType<number> | undefined => {
+    if (formValues) {
+      const initialValue = {
+        value: formValues.agency_id ?? formValues.first_nations_id,
+        label: '',
+        type: formValues.agency_id ? FundingSourceType.FUNDING_SOURCE : FundingSourceType.FIRST_NATIONS
+      } as IAutocompleteFieldOptionWithType<number>;
+
+      initialValue.label = `${findItemLabel(initialValue.value, initialValue.type)}`;
```

Comment:
Could simply move this inside the `initialValue` declaration

Reply:
I've updated this to keep the object creation altogether instead of splitting it up